common/scala/src/main/resources/reference.conf

```diff
@@ -28,6 +28,7 @@ whisk.spi {
   AuthenticationDirectiveProvider = org.apache.openwhisk.core.controller.BasicAuthenticationDirective
   InvokerProvider = org.apache.openwhisk.core.invoker.InvokerReactive
   InvokerServerProvider = org.apache.openwhisk.core.invoker.DefaultInvokerServer
+  DurationCheckerProvider = org.apache.openwhisk.core.scheduler.queue.NoopDurationCheckerProvider
```

Comment:
Regardless of whether the ansible changes make it in, I think we should merge in the changes to this file so that service can at least run with all of the necessary providers. If you want to put this file's changes in a separate PR I'd approve that. Don't have an opinion on the ansible change